test/utils/getCourtCaseById.ts

```diff
@@ -0,0 +1,10 @@
+import CourtCase from "../../src/services/entities/CourtCase"
+import getDataSource from "../../src/services/getDataSource"
+
+export const getCourtCaseById = async (caseId: number): Promise<CourtCase> => {
```

Comment:
There is a `getCourtCase` function that is doing something very similar, I wonder if we could modify/reuse that